semgrep-core/tests/OTHER/rules/entropy_python.yaml

```diff
@@ -0,0 +1,11 @@
+rules:
+  - id: entropy-python
+    patterns:
+      - pattern: $A = $B
```

Comment:
if you use $A = "$B" then in $B you should get the raw content of the string without the enclosing quotes.

Reply:
I was getting quotes because I was extracting the string data using the wrong function. I simplified it now, it's all good. I left a hack that performs some unescaping. It's not great but doing unescaping correctly will require a lot of work. I left a todo note about this in `AST_generic.ml`.